backend/src/tasks/test/__snapshots__/amass.test.ts.snap

```diff
@@ -1,29 +1,29 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`amass basic test: helpers.saveDomainsToDb 1`] = `
-Array [
+[
   Domain {
     "asn": 6762,
-    "discoveredBy": Object {
+    "discoveredBy": {
```

Comment:
Why did all these snapshots change?

Reply:
jest is not updated in this PR. could be changes to the db api from other commits. Not sure where they come from, but it seems to fix the tests.

Reply:
Yeah makes sense. Looks like this is happening on other PRs too. So let's update snapshots.